<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateZoneDialog</class>
 <widget class="QDialog" name="CreateZoneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Zone</string>
  </property>
  <layout class="QVBoxLayout" name="mZoneLayout">
   <item>
    <widget class="QGroupBox" name="mCommonOptionsBox">
     <property name="title">
      <string>Common Zone Options</string>
     </property>
     <layout class="QGridLayout" name="mCommonOptionsLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mNameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mZoneTypeLabel">
        <property name="text">
         <string>Zone Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="mZoneTypeBox"/>
      </item>
      <item row="4" column="1">
       <widget class="UtQtLatPosLineEdit" name="mLatitudeEdit"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="mHeadingLabel">
        <property name="text">
         <string>Heading</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mReferencePlatformLabel">
        <property name="text">
         <string>Reference Platform</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mLatitudeLabel">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mLongitudeLabel">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="UtQtLengthValueEdit" name="mMinimumAltitudeEdit"/>
      </item>
      <item row="7" column="1">
       <widget class="UtQtLengthValueEdit" name="mMaximumAltitudeEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fileLabel">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fileComboBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="mReferencePlatformBox"/>
      </item>
      <item row="5" column="1">
       <widget class="UtQtLonPosLineEdit" name="mLongitudeEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="mMinimumAltitudeLabel">
        <property name="text">
         <string>Minimum Altitude</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="mMaximumAltitudeLabel">
        <property name="text">
         <string>Maximum Altitude</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="UtQtAngleValueEdit" name="mHeadingEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mNameEdit"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="fillColorLabel">
        <property name="text">
         <string>Fill Color</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lineColorLabel">
        <property name="text">
         <string>Line Color</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="UtQtColorButton" name="mFillColorButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="UtQtColorButton" name="mLineColorButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGeometricOptionsBox">
     <property name="title">
      <string>Geometric Zone Options</string>
     </property>
     <layout class="QGridLayout" name="mGeometricOptionsLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mLateralAxisLabel">
        <property name="text">
         <string>Lateral Axis</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="UtQtLengthValueEdit" name="mLateralAxisEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mLongitudinalAxisLabel">
        <property name="text">
         <string>Longitudinal Axis</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="UtQtLengthValueEdit" name="mLongitudinalAxisEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mMinimumRadiusLabel">
        <property name="text">
         <string>Minimum Radius</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="UtQtLengthValueEdit" name="mMinimumRadiusEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mMaximumRadiusLabel">
        <property name="text">
         <string>Maximum Radius</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="UtQtLengthValueEdit" name="mMaximumRadiusEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mStartAngleLabel">
        <property name="text">
         <string>Start Angle</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="UtQtAngleValueEdit" name="mStartAngleEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mStopAngleLabel">
        <property name="text">
         <string>Stop Angle</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="UtQtAngleValueEdit" name="mStopAngleEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mPolygonalOptionsBox">
     <property name="title">
      <string>Polygonal Zone Options</string>
     </property>
     <layout class="QGridLayout" name="mPolygonalOptionsLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mAddPointsLabel">
        <property name="text">
         <string>Add Points</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="wkf::LocationSelector" name="mPointSelector">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTreeWidget" name="mTreeWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QGroupBox" name="mPointDataBox">
        <property name="title">
         <string>Point Values</string>
        </property>
        <layout class="QGridLayout" name="mPointDataLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="mPointLatLabel">
           <property name="text">
            <string>Latitude</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="UtQtLatPosLineEdit" name="mPointLatEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="mPointLonLabel">
           <property name="text">
            <string>Longitude</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="UtQtLonPosLineEdit" name="mPointLonEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="mAbsoluteButton">
        <property name="text">
         <string>Absolute</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="mRelativeButton">
        <property name="text">
         <string>Relative</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtLatPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLonPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtAngleValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLengthValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>wkf::LocationSelector</class>
   <extends>QPushButton</extends>
   <header>WkfSelectorWidgets.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtColorButton</class>
   <extends>QPushButton</extends>
   <header>UtQtColorButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
